### DISCUSSION TITLE
Looking for help with monorepo [yarn],[turborepo],[bare ignite app]

### DISCUSSION THREAD

**Discussion by mausic** [infinitered/ignite#2711] [Q&A]
Hey everyone!
I would be very happy to receive any help with an issue I am trying to solve for a few days. 
I receive strange infinite errors with a bare react native project setup.

Steps to reproduce:
1. Start a monorepo with `npx create-turbo@latest` select `yarn` as a package manager
2. `cd apps`
3. Create a bare RN project `npx ignite-cli@latest new pizza-app`
4. `cd pizza-app`
5. Create `index.js`, update `package.json` and `metro.config.js` according to [expo monorepo guidelines](https://docs.expo.dev/guides/monorepos/)
6. `yarn ios`
7. Observe infinite errors:
```bash
 ERROR  TypeError: Cannot assign to property 'username' which has only a getter, js engine: hermes
 ERROR  TypeError: Cannot assign to property 'username' which has only a getter, js engine: hermes
 ERROR  Invariant Violation: "main" has not been registered. This can happen if:
* Metro (the local dev server) is run from the wrong folder. Check if Metro is running, stop it and restart it in the current project.
* A module failed to load due to an error and `AppRegistry.registerComponent` wasn't called., js engine: hermes
 ERROR  Invariant Violation: No callback found with cbID 2 and callID 1 for module <unknown>. Args: '[{"app_state":"active"}]', js engine: hermes
 ERROR  Invariant Violation: No callback found with cbID 18 and callID 9 for module <unknown>. Args: '[{"app_state":"active"}]', js engine: hermes
 ERROR  Invariant Violation: No callback found with cbID 41 and callID 20 for module <unknown>. Args: '[0]', js engine: hermes
 ERROR  Invariant Violation: No callback found with cbID 45 and callID 22 for module <unknown>. Args: '[1]', js engine: hermes
 ERROR  Invariant Violation: No callback found with cbID 49 and callID 24 for module <unknown>. Args: '[2]', js engine: hermes
 ERROR  Invariant Violation: No callback found with cbID 63 and callID 31 for module <unknown>. Args: '[3]', js engine: hermes
 ERROR  Invariant Violation: No callback found with cbID 77 and callID 38 for module <unknown>. Args: '[4]', js engine: hermes
 ERROR  Invariant Violation: No callback found with cbID 81 and callID 40 for module <unknown>. Args: '[5]', js engine: hermes
 ERROR  Invariant Violation: No callback found with cbID 85 and callID 42 for module <unknown>. Args: '[6]', js engine: hermes
 ERROR  Invariant Violation: No callback found with cbID 89 and callID 44 for module <unknown>. Args: '[7]', js engine: hermes
 ERROR  Invariant Violation: No callback found with cbID 117 and callID 58 for module <unknown>. Args: '[8]', js engine: hermes
 ERROR  Invariant Violation: No callback found with cbID 123 and callID 61 for module <unknown>. Args: '[9]', js engine: hermes
...
```

Any advice or suggestions where to look for a fix?

upd: I am using `yarn 1.22.19`

**Comment by fpena** [comment]
Hello @mausic ! 

Thanks for creating this discussion. 

Can you please create an issue [here](https://github.com/infinitered/ignite/issues) instead? It would allow us to help and provide better assistance. 

Thanks again and looking forward to hearing back from you.

**Reply by mausic** [reply]
Hello @fpena !

Thanks for your reply. You can find an issue created here: https://github.com/infinitered/ignite/issues/2715

Thank you!

**Comment by fpena** [comment]
Based on [this comment](https://github.com/infinitered/ignite/issues/2715#issuecomment-2252766210), seems like the issue was resolved. 

Let us know if you have any other questions! Thanks.

**Reply by mausic** [reply]
I am not sure that this is a resolution. It's rather a crutch
